css/properties/border-image-repeat.json

```diff
@@ -0,0 +1,114 @@
+{
+  "css": {
+    "properties": {
+      "border-image-repeat": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-image-repeat",
+          "support": {
+            "chrome": {
+              "version_added": "15"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "15"
+            },
+            "firefox_android": {
+              "version_added": "15"
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "6"
+            },
+            "safari_ios": {
+              "version_added": "9.3",
+              "notes": "See <a href='https://bugs.webkit.org/show_bug.cgi?id=14185'>WebKit bug 14185</a>."
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "round": {
+          "__compat": {
+            "description": "<code>round</code>",
+            "support": {
+              "chrome": {
+                "version_added": "30",
+                "notes": "See <a href='https://crbug.com/169254'>Chromium bug 169254</a>."
```

Comment:
This links to a now-resolved bug in the behavior of the round subfeature. It seems like it might be relevant to keep information about when the behavior was buggy and when it wasn't, but I don't know how to correlate a Chromium commit with a released version of Chrome. Without that information, it seems like this is merely a historical note, like the iOS note above.

Reply:
Doesn't add value in my opinion. Let's remove this.